db/db-challenge3/db-008/db-008.sql

```diff
@@ -0,0 +1,8 @@
+select u.user_name as ユーザー名, c.chat_name as チャット名, substring(joined_at,1,10) as 参加日時
```

Comment:
（修正不要）

慣例ですが、asで日本語別名をつける際は、ダブルクォーテーションでくくるのが一般的です。
とはいえ、ダブルクォーテーションなしでも動作しますが。

```suggestion
select u.user_name as  "ユーザー名 ", c.chat_name as "チャット名 ", substring(joined_at,1,10) as "参加日時 "
```

Reply:
（要修正）

substringで参加日時のフォーマットを整えていますが、DATE_FORMATという関数があるのでそちらを利用しましょう。

substringで表現するよりも、直感的に何を表示したいかがわかりやすくなります。
（わかりやすい = 不具合を生みにくい、メンテナンスしやすい）